test/trackler/implementation_test.rb

```diff
@@ -59,6 +59,14 @@ def test_language_and_implementation_specific_readme
     assert_equal expected, implementation.readme
   end
 
+  def test_symlinked_file
+    problem = Trackler::Problem.new('fish', PATH)
+    implementation = Trackler::Implementation.new('animal', URL, problem, PATH)
+
+    expected = "This should get included in fish.\n"
+    assert_equal expected, archive_contents(implementation.zip, 'included-via-symlink.txt')
```

Comment:
Accessing it via the archive is redundant, you should be able to do: 
`assert_equal expected, implementation.files['included-via-symlink.txt']`